tests/test-cases/log-level-validation.yaml

```diff
@@ -31,7 +31,7 @@ tests:
     expect:
       exit_code: 1
       stderr:
-        - "Error: Invalid log level 'XTrace'. Valid options are: \\[Trace Debug Info Warning Off\\]"
+        - "Error: Invalid log level 'XTrace' set in the ATMOS_LOGS_LEVEL environment variable. Valid options are: \\[Trace, Debug, Info, Warning, Off\\]\\n"
```

Comment:
Note other tests are passing, and don't have the comma delimited list. Something seems off.

Reply:
see https://github.com/cloudposse/atmos/pull/957/files#diff-d81c64c201e1e391275c612d73add60a5a9898f0847ba457dbdaddcdda811e96R16

Reply:
Can you also test:
```
      - --logs-level=XTrace
```